### PR TITLE
Fix title of article to include 'and above' :cloud_with_lightning_and_rain:

### PR DESCRIPTION
**Pull request by blakedrumm**

### CODE REVIEW

File: iis/manage/configuring-security/how-to-set-up-ssl-on-iis.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+title: "How to Set Up SSL on IIS 7 and above"
 author: rick-anderson
 description: "The steps for configuring Secure Sockets Layer (SSL) for a site are the same in IIS 7 and above and IIS 6.0, and include the following: Get an appropriate ce..."
 ms.date: 11/22/2007
 ms.assetid: dbf635a5-921f-4488-99a2-bd1c2d1c0402
 msc.legacyurl: /learn/manage/configuring-security/how-to-set-up-ssl-on-iis
 msc.type: authoredcontent
 ---
-# How to Set Up SSL on IIS 7
+# How to Set Up SSL on IIS 7 and above
```
Comment (wadepickett) [inline iis/manage/configuring-security/how-to-set-up-ssl-on-iis.md:10]:
```suggestion
# How to Set Up SSL on IIS 7 or later
```

**Review by wadepickett — APPROVED** [review]
Approved, made a suggestion you can add.  Thanks!

**Comment by blakedrumm** [comment]
@wadepickett - When will the docs be updated?
title now

[https://learn.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis](https://learn.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis)

**Comment by wadepickett** [comment]
@John-Hart, I don't have rights to publish.  Could you publish this one?

**Comment by John-Hart** [comment]
@Rick-Anderson can you please merge this to live?

**Comment by wadepickett** [comment]
@Rick-Anderson, thanks!
@blakedrumm, your changes went live.

**Pull request merged**